crates/primitives/src/ids.rs

```diff
@@ -53,3 +53,4 @@ system_id!(SequenceId, u32);
 system_id!(IndexId, u32);
```

Comment:
These now need to implement Serialize, Deserialize, and SpacetimeType. They are going to pretend to be their wrapped types.